src/charm.py

```diff
@@ -461,7 +461,7 @@ def _on_add_admin_user_action(self, event: ActionEvent) -> None:
                 [
                     "bash",
                     "-c",
-                    f"./bin/bundle exec rake admin:create <<< $'{email}\n{password}\n{password}\nY'",
+                    f"./bin/bundle exec rake admin:create <<< $'{email}\n{password}\n{password}\nY'",  # pylint: disable=line-too-long
```

Comment:
Same comment here, this can be broken into multiple lines. black will do this automatically if you just insert two quotes where you want the line break to be

Reply:
So if you just write it like this: 

`"./bin/bundle exec rake admin:create <<< "f"$'{email}\n{password}\n{password}\nY'"`

`black` should automatically break this up for you